### PR TITLE
Add artifacts of scenario-1.3 to synapseConfigProject

### PR DESCRIPTION
**Pull request by dulanjalidilmi**
## Purpose
The purpose of this PR is to add and update to synapseConfigProject for scenario-1.3

## Approach
* Add artifacts of scenario-1.3
* Add util function to invoke api and get response to HTTPUtils
* Add util function to extract payload from a HTTP response to RESTClient

## Test environment
This is tested locally in following environment.
JDK - 1.8
OS - macOS High Sierra

### CODE REVIEW

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/HTTPUtils.java [hunk 1]
```diff
@@ -185,4 +188,21 @@ public static void invokeSoapActionAndCheckContains(String serviceUrl, String re
...
+    /**
+     * Invoke API and returns response
+     *
+     * @param url api invocation url
+     * @param request the request to transform
+     * @param contentType content-type of the request
+     * @return extracted payload from the returned http Response
+     * @throws IOException
+     */
+
```
Comment (milindaperera) [inline product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/HTTPUtils.java]: shall we remove the space

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/RESTClient.java [hunk 2]
```diff
@@ -242,6 +243,27 @@ public HttpResponse doDeleteWithPayload(String url, final Map<String, String> he
...
     }
 
+    /**
+     * Extracts the payload from a HTTP response
+     *
+     * @param httpResponse HttpResponse instance to be extracted
+     * @return Content payload
+     * @throws IOException If an error occurs while reading from the response
+     */
+    public String getResponsePayload(HttpResponse httpResponse) throws IOException {
```
Comment (milindaperera) [inline product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/RESTClient.java]: Let's move this to HTTPUtils

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/HTTPUtils.java (hunk 1 above)
Comment (dulanjalidilmi) [reply product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/HTTPUtils.java]: fixed.

File: product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/RESTClient.java (hunk 2 above)
Comment (dulanjalidilmi) [reply product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/RESTClient.java]: fixed.

**Pull request merged**